auto_round/autoround.py

```diff
@@ -1622,3 +1626,195 @@ def __init__(
 
 
 
+class AutoMLLMRound(AutoRound):
+    def __init__(
+            self,
+            model,
+            tokenizer,
+            bits: int = 4,
+            group_size: int = 128,
+            sym: bool = False,
+            layer_config: dict = None,
+            batch_size: int = 8,
+            amp: bool = True,
+            device: str = None,
+            lr_scheduler=None,
+            dataset: Union[str, list, tuple, torch.utils.data.DataLoader] = None,
+            extra_data_dir: Union[str, torch.utils.data.DataLoader] = None,
+            template: Union[str, Template] = None,
+            quant_vision: bool = False,
```

Comment:
better change a name, quant_nontext_module or something like that to extend to other modality later